### ISSUE TITLE
add more control over output from mcmc.diagnostics

### ISSUE THREAD

**Issue by martinamorris** [statnet/ergm#112]
in particular, only the plots (the most common alternate choice)

**Comment by krivit** [comment]
@martinamorris , @sgoodreau , what should the defaults be, then?

**Comment by sgoodreau** [comment]
Well, whenever we present these to folks in workshops we focused 100% on the plots.  At least I know I do, and I'm pretty sure @martinamorris and @drh20drh20 do the same.  Not sure about you, Pavel.  But in any case, that suggests that most of our user base will be focusing on the plots most of the time, so that seems like the best default to me.

**Comment by martinamorris** [comment]
i agree with SMG -- plots only as the default, possibly also the summary 
table showing the p-values for each term.
